sklearn/cluster/_optics.py

```diff
@@ -476,15 +477,21 @@ def compute_optics_graph(X, min_samples, max_eps, metric, p, metric_params,
 
     # Main OPTICS loop. Not parallelizable. The order that entries are
     # written to the 'ordering_' list is important!
-    # Note that this implementation is O(n^2) theoretically, but
-    # supposedly with very low constant factors.
+    # This implementation is O(n lg n) theoretically.
+
+    Heap = []
```

Comment:
please use lowercase names for local variables.

Reply:
sure